test/test_torch.py

```diff
@@ -16706,6 +16417,216 @@ def caller(cls,
     for test in tensor_op_tests:
         caller(cls, *test)
 
+def _generic_input_fn(dtype, device):
+    input = []
+    input.append(list(range(-5, 5)))
+    input.append([0 for x in range(-5, 5)])
+    input.append([x + 1e-6 for x in range(-5, 5)])
+    # Some vectorized implementations don't support large ranges
+    input.append([x + 1e10 for x in range(-5, 5)])
+    input.append([x - 1e10 for x in range(-5, 5)])
+    input.append(torch.randn(10).tolist())
+    input.append((torch.randn(10) + 1e6).tolist())
```

Comment:
Would multiplying by 1e6 be more interesting?

Reply:
Good idea, now there are no values between O(1) and O(1e10)